### PR TITLE
Visual Testing FAQ addition

### PR DESCRIPTION
**Pull request by kimsauce**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/visual/faq.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ For example, if content is loaded based on scrolling down to a particular sectio
 window.scroll(0, 1000);
 ```
 
-### Why is my screenshot not full-page?
+### Why is my screenshot not showing as a full page?
 
 Screener captures a full-page screenshot by default in all browsers. The height of the full-page screenshot is based on the scrollable height of the main document body.
```
Comment (eugenekatsov) [inline docs/visual/faq.md:32]: @kimsauce I'm sorry I may be a bit behind on wether or not we are removing references to Screener in favour of Sauce Visual. I used the latter in my change so wondering if it needs to be changed.

File: docs/visual/faq.md [hunk 2]
```diff
@@ -44,6 +44,11 @@ document.body.style.overflow = 'auto'
 ```
 
 
+### What happens if my page has broken HTML tags?
+
+Sauce Visual assumes that there are no broken HTML tags on your pages and will not check for this.
```
Comment (kimsauce) [inline docs/visual/faq.md:49]: Hi @eugenekatsov , when I looked at the source page, it said that the only change was the addition of this Q above. I wanted to make sure that was right!

File: docs/visual/faq.md (hunk 1 above)
Comment (kimsauce) [reply docs/visual/faq.md:32]: I can change that here! The Screener/Sauce Visual change is def something I wanted to address in the docs.

File: docs/visual/faq.md (hunk 2 above)
Comment (eugenekatsov) [reply docs/visual/faq.md:49]: yup, correct

File: docs/visual/faq.md (hunk 1 above)
Comment (eugenekatsov) [reply docs/visual/faq.md:32]: Great, if you can make the change, that'd be all. I'll approve once you are ready. Thank you!

Comment (kimsauce) [reply docs/visual/faq.md]: done!

**Review by eugenekatsov — APPROVED** [review]

**Pull request merged**